lib/application-processing/resolvers.ts

```diff
@@ -874,15 +880,54 @@ export const updateApplicationProcessingGenerateInvoice: Resolver<
   }
 
   // Generate application invoice
-  generateApplicationInvoicePdf(
+  const pdfDoc = generateApplicationInvoicePdf(
     application,
     session,
     // TODO: Remove typecast when backend guard is implemented
     application.applicationProcessing.appNumber as number,
     invoice.invoiceNumber
   );
 
-  return { ok: true };
+  // Upload pdf to s3
+  // file name format: PP-Receipt-P<YYYYMMDD>-<invoice number>.pdf
+  const fileName = `PP-Receipt-P${formatDateYYYYMMDD(invoice.createdAt).split('-').join('')}-${
+    invoice.invoiceNumber
+  }.pdf`;
+  const s3InvoiceKey = `rcd/invoices/${fileName}`;
```

Comment:
The example invoice rcd gave us had this format so i thought i might as well use it. I think it's better than `<invoiceNumber>.pdf`